README.md

```diff
@@ -1640,12 +1643,22 @@ Other Style Guides
 ## Variables
 
   <a name="variables--const"></a><a name="13.1"></a>
-  - [13.1](#variables--const) Always use `const` or `let` to declare variables. Not doing so will result in global variables. We want to avoid polluting the global namespace. Captain Planet warned us of that. eslint: [`no-undef`](https://eslint.org/docs/rules/no-undef) [`prefer-const`](https://eslint.org/docs/rules/prefer-const)
+  - [13.1](#variables--const) Always use `const` or `let` to declare variables. Not doing so will result in global variables in case strict mode is disabled. We want to avoid polluting the global namespace. Captain Planet warned us of that. eslint: [`no-undef`](https://eslint.org/docs/rules/no-undef) [`prefer-const`](https://eslint.org/docs/rules/prefer-const)
```

Comment:
All right.
Instead of _**in case strict mode is disabled**_ we can write _**in case strict mode is not enabled**_

Just tell me if we should rephrase it or simply remove this strict mode mention and keep it as it is?

Reply:
i think it'd be best to leave it as is; there's no value in pointing out that the engine will fix this for you in some cases, since the lint rule and guide just tell you to never do it. simple is better :-)